sdk/keyvault/Azure.Security.KeyVault.Keys/src/Azure.Security.KeyVault.Keys.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Description>This is the Microsoft Azure Key Vault Keys client library</Description>
     <AssemblyTitle>Microsoft Azure.Security.KeyVault.Keys client library</AssemblyTitle>
-    <Version>4.1.0-preview.1</Version>
+    <Version>4.0.1</Version>
```

Comment:
Does this make sense to decrement a version number?

Reply:
In the master branch, we use a "placeholder" until we either need to branch for servicing (which we don't in this case since nothing else is in flight for KV) or just "downgrade" - which it's really not since we haven't shipped - in master.

Besides, even if we did have to downgrade an actual release on nuget, this still works: 4.0.1 is a servicing release of 4.0.0, and still less than 4.1.0-preview.1, and as long as client's aren't opting into prereleases they'll get 4.0.1.

In meeting about this, we did bring up that it is a bit confusing that 4.1.0-preview.1 (if pre-existing) is newer according to semver rules and doesn't have 4.0.1 changes, but this is a universal versioning problem we can't really solve. Presumably, 4.1.0-preview.2 would and customers would just have to discover that based on the release notes...until we get our TARDIS fixed.